core/src/avm1/value.rs

```diff
@@ -29,13 +29,8 @@ pub enum Value<'gc> {
 }
 
 // This type is used very frequently, so make sure it doesn't unexpectedly grow.
-// On 32-bit x86 Android, it's 12 bytes. On most other 32-bit platforms it's 16.
-#[cfg(target_pointer_width = "32")]
 const _: () = assert!(size_of::<Value<'_>>() <= 16);
 
-#[cfg(target_pointer_width = "64")]
-const _: () = assert!(size_of::<Value<'_>>() == 24);
```

Comment:
Can you keep them separate, to be more explicit? Will be also useful if we ever do NaN boxing on 32-bit.